packages/synapse-react-client/src/components/CardDeck/CardDeck.Mobile.tsx

```diff
@@ -0,0 +1,53 @@
+import { CardDeckDataProps } from './CardDeck'
+import ExpandableContent from '../home_page/ExpandableContent'
+import LargeButton from '../../components/styled/LargeButton'
+import { ImageFileHandle } from '../widgets/ImageFileHandle'
+
+export default function CardDeckMobile({
+  //link,
+  description,
+  title,
+  color,
+  titleIconFileHandleAssociation,
+  ctaButtonText,
+  ctaButtonURL,
+}: CardDeckDataProps) {
+  const titleElement = (
+    <div className="Programs__Mobile__Header">
+      <span className="Programs__Card__header__icon">
+        {titleIconFileHandleAssociation && (
+          <ImageFileHandle
+            fileHandleAssociation={titleIconFileHandleAssociation}
+          />
+        )}
+      </span>
+      <span className="Programs__Mobile__Header__Title"> {title} </span>
+    </div>
+  )
+  const content = (
+    <div className="Programs__Mobile__Content">
+      {/* {link && (
+        <p className="Programs__Card__header__info__link">
+          <a
+            className="highlight-link"
+            href={link}
+            target="_blank"
+            rel="noreferrer"
+          >
+            Visit website
+          </a>
+        </p>
+      )} */}
```

Comment:
Why is this commented out? Planning to add it back later?

Reply:
This is commented out because I moved my in progress code to a different local branch while I worked on the css styling. This code corresponds to the header website link that is present in the original Programs component. I realized I needed to learn more about how the links are configured using the LinkConfig before implementing that.

Tldr: I will add this functionality back in. Just need to figure out how to do it.